frontend/src/tests/application-tests.js

```diff
@@ -142,19 +178,289 @@ export function applicationsTests({ apiGET, apiPOST }) {
         it.todo(
             "When submitting survey.js data attached files are stored on disk rather than as base64 strings in the database"
         );
-        it.todo(
-            "Can submit a jpg/png file as a 'transcript' for an application; the resulting file can be retrieved"
-        );
-        it.todo(
-            "Can submit a pdf file as a 'transcript' for an application; the resulting file can be retrieved"
-        );
-        it.todo(
-            "Can submit and retrieve multiple files as a 'transcript' for an application"
-        );
-        // This is to test for a possible regression related to https://github.com/rails/rails/issues/41903
-        it.todo("Can submit and retrieve attachments for a new application");
+        it("Can submit and retrieve attachments for a new application", async () => {
+            let resp = await apiPOST("/debug/active_user", taOnlyUser);
+            expect(resp).toHaveStatus("success");
+
+            let str = fs.readFileSync(
+                path.resolve(__dirname, "./image-data/dummy.txt"),
+                {
+                    encoding: "base64",
+                }
+            );
+            let txt_str = "data:text/plain;base64," + str;
```

Comment:
What is the significance of this prepend? Is that the default format of the files that are processed and stored on the backend? My main worry is that if someone submits a regular file that does not have this metadata, are we guaranteed to receive a file with this metadata after upload?

Reply:
This case tests using a `txt` file. When a user submits a text file, the part is prepended automatically into the string.